static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -17,16 +17,23 @@ interface BaseEvent {
   transaction: string;
 }
 
-export interface TimelineIssuePlatformEvent extends BaseEvent {
+interface TimelineIssuePlatformEvent extends BaseEvent {
+  'event.type': '';
```

Comment:
This is a new field we're going to start fetching. It just happens to always be the empty string!